### PR TITLE
Feature/cli syntax

### PR DESCRIPTION
**Pull request by fabiosantoscode**
This changes the syntax from `./times-components component-add` to `./times-components add component`. I had a lot of trouble making this work with a proper usage message (since subcommand doesn't support nested subcommands), but ended up modifying process.argv to simplify the process.

### CODE REVIEW

**Comment by trevorah** [comment]
But why?

**Comment by fabiosantoscode** [comment]
@trevorah I had a request from @craigbilner to change the CLI syntax to use spaced words in commands

**Comment by craigbilner** [comment]
@trevorah following a CLI pattern used by most other libs a la kubectl, Ember, Angular etc.

File: lib/cli/templates.js [hunk 1]
```diff
@@ -1,96 +1,47 @@
...
 `);
 
-const storyFile = handlebars.compile(`
-import React from 'react';
-import { storiesOf } from '@storybook/react';
-import {{{component}}} from './{{{packageName}}}';
+const storyFile = handlebars.compile(`import "react-native";
+import React from "react";
+import { storiesOf } from "@storybook/react-native";
+import {{{component}}} from "./{{{packageName}}}";
```
Comment (craigbilner) [inline lib/cli/templates.js]: Can this import `TestComponent` from `test-component`?

**Comment by rudids** [comment]
@craigbilner @trevorah are we merging?

**Pull request merged**